src/os/bluestore/BlueStore.cc

```diff
@@ -4941,19 +4941,19 @@ void BlueStore::_init_logger()
 		 "r_l", PerfCountersBuilder::PRIO_CRITICAL);
   b.add_time_avg(l_bluestore_read_onode_meta_lat, "read_onode_meta_lat",
     "Average read onode metadata latency",
-    "roml", PerfCountersBuilder::PRIO_USEFUL);
+    "or_l", PerfCountersBuilder::PRIO_USEFUL);
   b.add_time_avg(l_bluestore_read_wait_aio_lat, "read_wait_aio_lat",
     "Average read latency",
-    "rwal", PerfCountersBuilder::PRIO_USEFUL);
+    "riol", PerfCountersBuilder::PRIO_USEFUL);
   b.add_time_avg(l_bluestore_compress_lat, "compress_lat",
     "Average compress latency",
-    "coml", PerfCountersBuilder::PRIO_USEFUL);
+    "cc_l", PerfCountersBuilder::PRIO_USEFUL);
```

Comment:
why cc?

Reply:
What about `cprl`, with `cpr` referring to compress, and `l` referring to latency? `c_l` is already taken by `bluestore_commit_lat`-- otherwise I would choose that.

Reply:
I like the first pairing.